### PR TITLE
[Tests only] Add acceptance test to export created mounts using occ command

### PR DESCRIPTION
**Pull request by jasson99**
## Description
 Add acceptance test to export created mounts using occ command

## Related Issue

- Related #36704 

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -1225,6 +1225,36 @@ public function theFollowingLocalStoragesShouldNotExist(TableNode $mountPoints)
...
+			$isStorageEntryListed = false;
+			foreach ($commandOutput as $listedStorageEntry) {
+				if ($expectedStorageEntry["MountPoint"] === $listedStorageEntry->mount_point) {
+					Assert::assertEquals($expectedStorageEntry['Storage'], $listedStorageEntry->storage);
+					Assert::assertEquals($expectedStorageEntry['AuthenticationType'], $listedStorageEntry->authentication_type);
+					Assert::assertStringStartsWith($expectedStorageEntry['Configuration'], $listedStorageEntry->configuration);
+					Assert::assertEquals($expectedStorageEntry['Options'], $listedStorageEntry->options);
+					Assert::assertEquals($expectedStorageEntry['ApplicableUsers'], $listedStorageEntry->applicable_users);
+					Assert::assertEquals($expectedStorageEntry['ApplicableGroups'], $listedStorageEntry->applicable_groups);
+					$isStorageEntryListed = true;
```
Comment (dpakach) [inline tests/acceptance/features/bootstrap/OccContext.php]: Just break the loop after this, no point in keep going.

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 2]
```diff
@@ -1225,6 +1225,36 @@ public function theFollowingLocalStoragesShouldNotExist(TableNode $mountPoints)
...
+				}
+			}
+			if ($isStorageEntryListed === false) {
+				throw new Exception("Expected local storages not found");
```
Comment (dpakach) [inline tests/acceptance/features/bootstrap/OccContext.php]:
```suggestion
				throw new Exception("Expected local storage: $expectedStorageEntry not found in the command output");
```

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 3]
```diff
@@ -1251,6 +1281,16 @@ public function administratorDeletesFolder($folder) {
 		$this->invokingTheCommand('files_external:delete --yes ' . $mount_id);
 	}
 
+	/**
+	 * @When the administrator exports the mounts using the occ command
```
Comment (dpakach) [inline tests/acceptance/features/bootstrap/OccContext.php]:
```suggestion
	 * @When the administrator exports the local storage mounts using the occ command
```

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 1 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/OccContext.php]: fixed

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 2 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/OccContext.php]: fixed

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 3 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/OccContext.php]: fixed

**Comment by phil-davis** [comment]
Needs #36781 to be merged first, then rebase this

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 4]
```diff
@@ -1272,6 +1272,37 @@ public function theFollowingLocalStorageShouldBeListed(TableNode $table) {
...
+					$isStorageEntryListed = true;
+					break;
+				}
+			}
+			if ($isStorageEntryListed === false) {
+				throw new Exception("Expected local storage: $expectedStorageEntry not found");
+			}
+		}
+	}
+
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: This is duplicated - delete it

**Review by phil-davis — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 4 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/OccContext.php]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**